config/MasterTranslations.xml

```diff
@@ -1866,16 +1866,17 @@ The course is saved automatically on closing of the editor and overrides the sel
 			<Text language="de"><![CDATA[
 
 Mit Courseplay ist es möglich, Feldkurse mit zusätzlichen Funktionen zu generieren, beispielsweise Vorgewende.
-Ebenso ermöglicht es die Nutzung von Ballenpressen und Sammelwagen mit dem Helfer auf demselben Kurs, welcher zuvor von einem Mähwerk oder Drescher bearbeitet wurde.
+Ebenso ermöglicht es die Nutzung von Ballenpressen und Sammelwagen mit dem Helfer auf dem selben Kurs, welcher zuvor von einem Mähwerk oder Drescher bearbeitet wurde.
 Eine weitere Funktion ist das Ballenwickeln oder -sammeln auf einem Feld.
 Ganz neu in Courseplay ist die Bearbeitung von Reben.
```

Comment:
Ändern in "Bearbeitung von Reisfeldern"? (Reben sind ja nun nicht mehr "ganz neu")